.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: node_js
+services:
```

Comment:
Please leave a new line after each config section. It makes it easier to follow

Reply:
Resolve as requested